src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -304,6 +304,23 @@ protected static void assertEqualsDoubleArray(final double[] actual, final doubl
             Assert.assertEquals(actual[i],expected[i],tolerance,"array position " + i);
     }
 
+    /**
+     * Checks whether two long arrays contain the same values or not.
+     * @param actual actual produced array.
+     * @param expected expected array.
+     * @param tolerance maximum difference between double value to be consider equivalent.
+     */
+    protected static void assertEqualsLongArray(final long[] actual, final long[] expected) {
+        if (expected == null)
+            Assert.assertNull(actual);
+        else {
+            Assert.assertNotNull(actual);
+            Assert.assertEquals(actual.length,expected.length,"array length");
+        }
+        for (int i = 0; i < actual.length; i++)
+            Assert.assertEquals(actual[i],expected[i],"array position " + i);
```

Comment:
could you include the value found at the position in both the actual and expected in the error message